### ISSUE TITLE
Horizontal stripes seen on Kindle Fire HD 7 (2013)

### ISSUE THREAD

**Issue by proski** [geometer/FBReaderJ#306]
The stripes are most noticeable after the last line of a paragraph, aligned with the bottom of the text. I put purple lines on the screenshot on those stripes. Look to the right of the purple lines. There are repeating groups of darker pixels. It may seem very minor on the computer screen, but it's easy to notice on the Kindle Fire screen and it gets annoying when reading.
![screenshot_2015-03-09-21-50-00-stripes](https://cloud.githubusercontent.com/assets/1317472/6568118/ade165a6-c6a8-11e4-9a5d-220eda3bee2b.png)

**Comment by liquiddandruff** [comment]
It's a case of tiled textures. Personally I don't like the parchment paper background anyways (looks cheesy to me) so I read exclusively in night mode.

**Comment by proski** [comment]
I'm fine with a texture as long as it's rendered without artifacts. Besides, it's the default. Users should not be expected to change it right away. It looks good on other devices. This texture is called Sepia. I found a file called `assets/wallpapers/sepia.jpg` in the sources and it doesn't have those stripes. I suspect it's an error that causes that file to be scaled. By the way, the artifacts for "Hard Paper" are even more pronounced.

**Comment by proski** [comment]
Android (or at least Android as instructed by FBReader) doesn't render JPG correctly. I made a simple pattern, saved it at the highest quality to a JPG file and used it as the background for FBReader. The PNG file is the pattern cut from the screenshot. There are 5 blurred areas for 40 pixels, i.e. blurred areas repeat every 8 pixels. Both Kindle Fire HD 7 with FireOS and Samsung Galaxy S4 with CyanogenMod 11 exhibit the same blurring, but it's probably easier to see on Kindle Fire. Perhaps FBReader should use PNG. PNG patterns don't bleed.
![pattern](https://cloud.githubusercontent.com/assets/1317472/6588905/e0e4e0ba-c76e-11e4-9262-373ac0937cc0.jpg)
![kindle-pattern](https://cloud.githubusercontent.com/assets/1317472/6588907/e417bf1e-c76e-11e4-99c4-34bcddc1b484.png)

**Comment by proski** [comment]
It looks like the `sepia.jpg` file included with the sources already had certain blurring. Also, there are some subtle horizontal lines in it. So I suggest that we convert that pattern to PNG and turn it 90 degrees to make the lines vertical. I set it up as my background and I really like it. Vertical lines don't distract the eyes from the text as much as the horizontal ones.
![sepia](https://cloud.githubusercontent.com/assets/1317472/6589224/00c1f7de-c773-11e4-8bc7-233e09b9f8d8.png)

**Comment by liquiddandruff** [comment]
Nice work! Is that the image that produced your desired results? If so, you should make it a commit. I can do that for you if you want.

**Comment by proski** [comment]
Yes, that's the image I used for background. Just a few points to check before making the commit. Is it possible to render JPG with the high (i.e. actual) quality? If yes, the PNG conversion won't be needed (although it could still be considered). Instead, the JPG file could be losslessly rotated. Is it possible to replace a JPG file with a PNG file in the resources? I don't have the Android development tools and it will take me time to install and learn then to test it. It looks like the original file already has some blurry bands (although it's hard to tell for 40x40 pixels). Is there a way to find the original pattern and convert it to PNG?
